recipes/gneiss/meta.yaml

```diff
@@ -0,0 +1,33 @@
+package:
+  name: gneiss
+  version: 0.1.0
+
+source:
+  git_url: https://github.com/biocore/gneiss.git
+
+
+requirements:
+  build:
+    - python
+    - scikit-bio ==0.4.2
```

Comment:
Okay. It is unlikely scenario on real trees as the bug arises when pruning where a single descendent stems from the root, but worth playing safe especially if none of the 0.5 machinery is in use